pykickstart/commands/liveimg.py

```diff
@@ -39,8 +39,8 @@ def __eq__(self, other):
             return False
 
         return self.url == other.url and self.proxy == other.proxy and \
-               self.noverifyssl == other.noverifyssl and \
-               self.checksum == other.checksum
+            self.noverifyssl == other.noverifyssl and \
+            self.checksum == other.checksum
```

Comment:
I like all the conditions lined up to the right of the return here.  That occurs a handful of other places in this mega patch, too.  Aside from that everything looks great to me.